.github/workflows/playwright.yml

```
@@ -77,10 +62,10 @@ jobs:
         retention-days: 7
 
-    # - name: Deploy report to GitHub Pages
-    #   if: always()
-    #   uses: peaceiris/actions-gh-pages@v3
-    #   with:
-    #     # # github_token: ${{ secrets.GITHUB_TOKEN }}
-    #     # publish_dir: sinatra/tests/playwright-report
-    #     # publish_branch: gh-pages
+    - name: Deploy report to GitHub Pages
+      if: always()
+      uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)